### PR TITLE
Doc 23 upd readme

### PR DESCRIPTION
**Pull request by raproid**
What I did:

- Shaped up the documentation and fixed some things.

How I did it:
- Changed README.md to conform with the requirements from the team.
- Fixed names of panels, panel description on the Grafana dashboard. 
- Removed the dead panels from the dashboard.
- Added the correct `addr` and `url` params into `observer.go`.

### CODE REVIEW

File: scripts/monitor/monitor.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+ #!/usr/bin/env bash
```
Comment (pavel-moskovkin) [inline scripts/monitor/monitor.sh:1]: why do we need this change?

Comment (raproid) [reply scripts/monitor/monitor.sh]: it is a very !not! important change.

Comment (raproid) [reply scripts/monitor/monitor.sh]: fixed

File: configuration/observer.go [hunk 2]
```diff
@@ -73,9 +73,9 @@ func (Observer) Default() *Observer {
 			CacheSize:           10000,
 			Auth: Auth{
 				Required:      true,
-				URL:           "https://{access_token_url}",
-				Login:         "{unique_login}",
-				Password:      "{strong_password}",
```
Comment (LinnikD) [inline configuration/observer.go:78]: ${unique_login} I think it's better to keep placeholders and not default values

Comment (raproid) [reply configuration/observer.go:78]: Is "<your_login>" a correct placeholder in this case?
Or does the language enforce this type of syntax ${your_login}?

Comment (raproid) [reply configuration/observer.go:78]: Made them empty according to the resolution found in our Slack conversation.

**Pull request merged**